app/forms/idv/document_capture_form.rb

```diff
@@ -31,6 +36,12 @@ def back_image_or_image_data_url_presence
       errors.add(:back_image, :blank)
     end
 
+    def selfie_image_or_image_data_url_presence
+      return unless FeatureManagement.liveness_checking_enabled?
```

Comment:
@jmhooper so I added this, but I don't see a convenient way to check that either there is no SP or the SP requires liveness checking. This is just a model with no access to the session or any of our current helpers. Open to suggestions...

Reply:
Maybe flag on the initializer?